package.json

```diff
@@ -128,12 +135,7 @@
   "ember": {
     "edition": "octane"
   },
-  "dependencies": {
-    "c3": "0.7.20",
-    "d3": "6.6.0",
-    "d3-geo": "^3.1.0",
-    "date-fns": "^2.29.3",
-    "postscribe": "^2.0.8",
-    "topojson": "^3.0.2"
-  }
+  "pre-commit": [
+    "lint"
```

Comment:
How does this work exactly? How does git know it should run this before committing?

Also, while I think it's fine, I feel this might be controversial so not sure if we can roll this out before it's discussed 😬 (I don't make the rules, sorry).

Reply:
It uses an npm package! https://www.npmjs.com/package/pre-commit


That is fair! I can turn this into a draft for the time being?

Reply:
> It uses an npm package! https://www.npmjs.com/package/pre-commit

Oh, nice, I didn't see it when I glanced at the other changes, that makes sense 👍.

> That is fair! I can turn this into a draft for the time being?

I'm not sure. If everyone on the team is on board, feel free to keep it, but be warned that there might be complaints in the future 😄. I wouldn't let it block the other changes though. So we either just go with it and roll back if needed, or split it to a separate PR.

Reply:
I guess it's a "do you prefer to know immediately or do you prefer to know when the CI fails" 😭

Reply:
Let's just do it. We'll deal with the fallout when it happens 😄.